xml/signals/BR-2003/appearance-2-h-sema.xml

```diff
@@ -35,33 +35,35 @@
         <date>2011-08-04</date>
         <authorinitials>KD</authorinitials>
         <revremark>Addition of Specific Appearances and aspect mapping</revremark>
-    </revision>  
+    </revision>
   </revhistory>
-  
+
   <aspecttable>BR</aspecttable>
 
   <name>Semephore Home Signal</name>
-  
+
   <description>This uses a single head to control the appearance</description>
-  
+
   <appearances>
 
     <appearance>
       <aspectname>Proceed</aspectname>
       <show>green</show>
       <imagelink type="large">../../../resources/icons/BritishRailSignals/Large/semaHomeClear.gif</imagelink>
       <imagelink type="aspects">../../../resources/icons/BritishRailSignals/Aspects/Proceed.gif</imagelink>
+      <imagelink type="medium">../../../resources/icons/BritishRailSignals/Medium/semaHomeProceed.gif</imagelink>
     </appearance>
 
     <appearance>
       <aspectname>Danger</aspectname>
       <show>red</show>
       <imagelink type="large">../../../resources/icons/BritishRailSignals/Large/semaHomeDanger.gif</imagelink>
       <imagelink type="aspects">../../../resources/icons/BritishRailSignals/Aspects/Danger.gif</imagelink>
+      <imagelink type="large">../../../resources/icons/BritishRailSignals/Medium/semaHomeDanger.gif</imagelink>
```

Comment:
should this be <imagelink type="medium">  ?

Reply:
Thanks for catching that!  Fixed.